spec/swagger.yaml

```diff
@@ -0,0 +1,82 @@
+swagger: '2.0'
+schemes:
+    - https # Remove if your API donsn't support HTTPS
+host: citrination.com
+basePath: /api
+info:
+    title: Citrination
+    version: 1.0.0
+    termsOfService: 'https://citrination.com/tos'
+    contact:
+        name: Citrine Informatics
+        email: admin@citrine.io
+        url: 'https://citrine.io'
+    license:
+        name: Apache License Version 2.0
+        url: 'https://www.apache.org/licenses/LICENSE-2.0'
+    description: |
+        # Introduction
+        The Citrination API provides much of the functionality that is available through the Citrination web interface. In addition, there exists some functionality that is only available through the API (for example, some of the more advanced query elements). All access to the API is controlled using a unique token assigned to each user (see the section on authentication).
+
+        Users may access the API directly through https requests, or may use one of the client libraries described below.
+
+        # Python client
+        A python client is available through [github](https://github.com/CitrineInformatics/python-citrination-client) or can be installed with pip using `pip install citrination-client`. There may be issues installing libraries that depend on [six](https://pypi.python.org/pypi/six) on OSX. If you encounter issues during installation, you may also try `pip install --ignore-installed six citrination-client`. The python client can be configured using:
+
+        ```python
+        from citrination-client import *
+
+        client = CitrinationClient('YOUR_API_KEY')
+        ```
+
+        If you are using a private instance of Citrination (e.g. https://mysite.citrination.com), then your client can be configured with:
+
+        ```python
+        from citrination-client import *
+
+        client = CitrinationClient('YOUR_API_KEY', 'https://mysite.citrination.com')
+        ```
+
+        Note that all variables and method names are represented in snake case in the python client library, rather than camel case as is used in this documentation. For example, a variable named `updatedAt` in this documentation would be `updated_at` in the python library.
```

Comment:
Could we draw more attention to this line, e.g. with a bold **Note**.  It is a common source of confusion.

Radical proposal: most of our external users are in python, so we could use the python notation as the default.

Reply:
`Could we draw more attention to this line, e.g. with a bold Note. It is a common source of confusion.` ... adding

`Radical proposal: most of our external users are in python, so we could use the python notation as the default.` ... Sorry, but no. This should document the actual API. _However_, I don't have any yet, but we can include examples in python to illustrate these differences. Here is an example of documentation that does that: https://rebilly.github.io/RebillyAPI/#tag/API-Keys%2Fpaths%2F~1api-keys%2Fpost. Note the ability to switch between JSON and PHP under Request Samples on the right section.